code/src/WijmoProvider/Features/ConditionalFormat.ts

```diff
@@ -58,13 +71,18 @@ namespace WijmoProvider.Feature {
             case Rules.LessThan:
                 return cellValue < comparedValue;
             case Rules.BeginsWith:
-                return cellValue.startsWith(comparedValue);
+                return cellValue && cellValue.startsWith(comparedValue);
```

Comment:
Do all cases require the cellValue to exist? Should you have a condition above of the switch?

Reply:
As explained to you privately, this won't be done.